### ISSUE TITLE
Is exit code always 0?

### ISSUE THREAD

**Issue by spinus** [ameingast/postgresql-simple-migration#7]
I started using this program to do migrations as it's very simple to use and does the job, thank you for this.

I run migrations on my deployment phase and I just changed one of existing migrations to see what happens. Nothing happened, my deployment went as usual. I was surprised that validation didn't throw an error. After checking logs (and adding some additional error logs to show exit codes) I found that program returns 0 exit code even if there are any errors:

```
# validate
Sep 05 00:58:42 dev db-start[6489]: Checksum mismatch:        0001.sql
Sep 05 00:58:42 dev db-start[6489]: 0
# migrate
Sep 05 00:58:42 dev db-start[6489]: Fail:        0001.sql
Sep 05 00:58:42 dev db-start[6489]: 0
```

Would it be possible to change exit code in case of errors? In other way I have to parse the output which is probably not the best way to communicate problems to the other process.

**Comment by ameingast** [comment]
Thanks for the bug report. 

The standalone application now exits with 1 for failing migrations.

It's fixed and released in [0.1.7.0](https://hackage.haskell.org/package/postgresql-simple-migration-0.1.7.0).

**Comment by spinus** [comment]
Wow, that was extremely quick. Thank you very much @ameingast!
